tutorials/introduction/06-kolmogorov_equations.jmd

```diff
@@ -0,0 +1,56 @@
+---
+title: Kolmogorov Equations
+author: Ashutosh Bharambe
+---
+
+
+```julia
+using Flux, StochasticDiffEq
+using  NeuralNetDiffEq
+using Distributions
+using Plots
+using LinearAlgebra
+```
+
+
+```julia
+g(u , p , t) = 0.5
+f(u , p , t) = -u
+d = 1
+sdealg = EM()
+ensemblealg = EnsembleThreads()
+u0 = Normal(0.00 , 0.1)
+xspan = (-3.0 , 3.0)
+tspan = (0.0 , 1.0)
+prob = SDEProblem(f , g , u0 , tspan ; xspan = xspan , d = d)
+opt = Flux.ADAM(0.05)
+m = Chain(Dense(1, 16, elu),Dense(16, 64, elu), Dense(64 , 5 , elu) , Dense(5 , 1))
+m = fmap(f64 , m)
+sol = solve(prob, NNKolmogorov(m,opt , sdealg,EnsembleDistributed()) , verbose = true, dt = 0.1,
+            abstol=1e-10, dx = 0.001 , trajectories = 50000 ,  maxiters = 500 )
+```
+
+
+```julia
+T = tspan[2]
+mu = 1
+sig = 0.5
+sig2 = (sig^2 /(2*mu))exp(-2*mu*T)*(exp(2*mu*T) - 1)
+analytical(xi) = pdf.(Normal(0.00 , sqrt(sig2)) , xi)
```

Comment:
are you sure about the scaling on the input here? IIRC it was supposed to be the variance.

Reply:
Where is the input scaled? Its sigma^2 and then I have taken its sqrt in Normal Pdf.

Reply:
![Screen Shot 2020-06-10 at 12 19 35 PM](https://user-images.githubusercontent.com/43771652/84236123-bf930380-ab14-11ea-938a-f17766bde987.png)
This is supposed to be the pdf right?

Reply:
With Distributions.jl you're supposed to do Normal(mean,variance) IIRC, and you're passing Normal(mean,std)?